src/callproc.c

```diff
@@ -52,6 +52,8 @@ along with GNU Emacs.  If not, see <http://www.gnu.org/licenses/>.  */
 #include "nsterm.h"
 #endif
 
+int rust_make_temp(char *template, int flags, int *errorcode);
```

Comment:
This should be on a header file (e. g.: `rust-temp.h`) to avoid code duplication.